src/admin/containers/labels.js

```diff
@@ -33,18 +32,25 @@ const lines =
   , 'address3'
   , 'address4'
   , 'postcode'
-  , 'county'
+  , 'deliverer'
   ]
 
 
 const font_style = text =>
   ({ style: { fontSize: Math.min(12, 300 / text[0].length) + 'pt' } })
 
+
+const add_deliverer = arr =>
```

Comment:
The pointfree version would look something like

`const add_deliver = flip(chain)(flip(over)(defaultTo('POST')), compose(lensIndex, add(-1), length))`

I think it's more declarative the way I wrote. I like to avoid messing with indices -- particularly negative indices -- as much as possible. You also don't duplicate the array twice, as one does with slicing and spreading, but that's less important.